src/base/async/int.c

```diff
@@ -1674,7 +1674,9 @@ void redirect_devices(void)
  */
```

Comment:
More explicitly it would be 'Change to pass redirector **version** not DOS version'